### PR TITLE
Adding frontends

### PR DESCRIPTION
**Pull request by gburges**

### CODE REVIEW

File: src/components/ClowdApp.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 //import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
-import { K8sResourceCommon} from '@openshift-console/dynamic-plugin-sdk';
+import { K8sResourceCommon, useK8sWatchResources} from '@openshift-console/dynamic-plugin-sdk';
```
Comment (psav) [inline src/components/ClowdApp.tsx:3]: This is already imported in the line below. Is there a need for it to be imported again?

**Review by psav — DISMISSED** [review]
I was envisaging a new nav entry for Frontends that listed them all - not including them in the App table. There may be some apps without frontends, and some frontends without apps.

File: src/components/ClowdApp.tsx (hunk 1 above)
Comment (gburges) [reply src/components/ClowdApp.tsx:3]: probably not needed, it was add by the yarn build probably

**Review by adamrdrew — APPROVED** [review]
LGMT!!!!!

**Pull request merged**